airbyte-integrations/connectors/source-snowflake/acceptance-test-config.yml

```diff
@@ -10,9 +10,9 @@ acceptance_tests:
     tests:
       - config_path: "secrets/config.json"
         status: "succeed"
-  discovery:
-    tests:
-      - config_path: "secrets/config.json"
+  #  discovery:
```

Comment:
don't comment. Just remove. It'll still be in github for posterity

Reply:
Is this test safe to remove, though? Snowflake is certified.

Reply:
@postamar Interesting, are you sure Snowflake source is certified? I didnt know it was certified! Also this is failing for discovery! I have not even touched any piece of code related to discover call so certainly not related to my changes!

Reply:
I don't think snowflake source is officially certified. @bleonard did some work to pass the snowflake tests to become a partner, but I don't think we officially moved this connector to certified status.
@subodh1810 let's keep the test commented like you did, and I'll create a github issue to remind ourselves to fix this